packages/web-components/src/components/pictogram-item/pictogram-item.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html, internalProperty, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings';
+import styles from './pictogram-item.scss';
+import '../content-item/content-item';
+import '../content-item/content-item-heading';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ *  Pictogram item.
+ *
+ * @element dds-pictogram-item
+ */
+@customElement(`${ddsPrefix}-pictogram-item`)
+class DDSPictogramItem extends LitElement {
+  /**
+   * Internal Property for get copy from slot
+   */
+  @internalProperty()
+  copy = '';
+
+  protected _handleSlotChange({ target }: Event) {
+    const { name } = target as HTMLSlotElement;
+    if (name === 'copy') {
+      this.copy = ((target as HTMLSlotElement).assignedNodes()[0] as HTMLElement).innerText
+        .replace(/[\n\r]+|[\s]{2,}/g, ' ')
+        .trim();
+    }
+  }
```

Comment:
Any reason mangling the text content of a slotted child instead of making `copy` a public attribute/prop and let user use it instead?

Reply:
I thought it would be better to receive it as a `slot`. We are receiving content this way for most of our web-components.

Reply:
Good point, we have had a exception, though; Copy content containing markdown doesn't use `<slot>` as a similar reason as here.

If you feel very strongly about "every content should be `<slot>`", then one suggestion is using `textContent` instead of `innerText`. In this way, you need less mangling (or no mangling is needed).

Reply:
@asudoh this was addressed on my latest commit, but the replace and trim are still necessary, otherwise, the text looks off, similar when used within a `code` tag